oaknational/Oak-Web-Application
# Chore/upgrade mux [LESQ-822]

## Description

Music year: 2011

- Updates mux-node and mux-player-react to latest versions
- update syntax for fetching signed urls
- add required secrets to google secret manager

## How to test

1. Go to lesson pages, eg https://deploy-preview-2417--oak-web-application.netlify.thenational.academy/teachers/programmes/chemistry-secondary-ks4-foundation-edexcel/units/atomic-structure-and-the-periodic-table/lessons/atomic-structure-negligible-electron-mass#video
2. The video should play as usual
